### DISCUSSION TITLE
ERROR: Uncaught ReferenceError: EffectMaterial is not defined at swiper-test.html:301:19

### DISCUSSION THREAD

**Discussion by bleeckerj** [nolimits4web/swiper#7599] [Q&A]
Hello!

Using Swiper Studio — so I bought the premium thing. I am trying to arrange the Material-style Swiper within a large page of content. When I wrap it in any <div> it becomes unusable — in fact, it disappears from view. Not sure why this is, so clearly something I am not understanding.

Additionally, when I use the HTML 5 code verbatim exported from the Swiper Studio, I see this error in the browser console: "Uncaught ReferenceError: EffectMaterial is not defined at swiper-test.html:301:19"

The snippet from this line looks like this:

```
<script>
      var swiper = new Swiper(".swiper", {
        modules: [EffectMaterial],
        slidesPerView: 2,
        spaceBetween: 16,
        effect: "material",
      });
    </script>
```

As mentioned, this is the code exported from Swiper Studio verbatim, except for adjusting one path to the CSS.

Anyone else have this issue? I could be doing something wrong — but then again, I'm not doing much other than putting this into a clean test HTML page as is.

Needless to say, the swiper doesn't work at all. Just a static image appears on the page.

Julian

**Comment by nolimits4web** [comment]
Please use support on Swiper Studio

**Comment by bleeckerj** [comment]
Please tell me where that is? I emailed the support email on the site and
received a message that the email address was blocked.




Julian Bleecker, BSEE, MSEng, Ph.D.
https://nearfuturelaboratory.com
https://shop.nearfuturelaboratory.com
https://julianbleecker.com
https://omata.com

**Reply by nolimits4web** [reply]
Yes via email to studio@swiperjs.com, we don’t block any email addresses and we can not. Try sending from another email address
